core/tests/test_utils.py

```diff
@@ -3697,13 +3731,15 @@ def save_new_topic(
         Returns:
             Topic. A newly-created topic.
         """
+        canonical_story_ids = canonical_story_ids or []
+        additional_story_ids = additional_story_ids or []
```

Comment:
setting default value to `[]` would work?

Reply:
@DubeySandeep It would work, but I heard that it's not a good practice in Python to use mutable objects as default values within a function's parameters. It can lead to confusion when those objects get mutated.